dashboard/app/views/pd/workshop_enrollment/new.html.haml

```diff
@@ -16,6 +16,17 @@
         = render partial: 'workshop_details'
         %h2
           Your Information
+        - unless @signed_in
+          %div.info-box-teal
+            %div.left
+              =image_tag image_url("info.png")
```

Comment:
I wonder if we should have a more specific filename than `info`.  Something like `info_icon` maybe?